src/css/imports/_navigation-collapsible.css

```diff
@@ -0,0 +1,151 @@
+/* NAV WRAPPER */
```

Comment:
This file needs general cleanup (extra line breaks, comments are sort of scattered and need to be made uniform. Labeling sections of this file isn't necessary, using clear class naming should provide that clarity.

Make comments in the following format: